src/main/kotlin/no/nav/tilleggsstonader/kontrakter/søknad/barnetilsyn/AvsnittBarnetilsyn.kt

```diff
@@ -26,10 +27,17 @@ data class BarnMedBarnepass(
     val navn: TekstFelt,
     val ident: TekstFelt,
     val type: EnumFelt<TypeBarnepass>,
+    val utgifter: Utgifter?,
```

Comment:
Hvis vi setter denne til default null stopper det ikke andre for å ta den i bruk uten at vi har oppdatert versjon i sak/soknad-api

```suggestion
    val utgifter: Utgifter? = null, // fjern default null når denne er tatt i bruk rundt om
```